### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by george124816**
# Contributor checklist

- [ ] Bug fixes include regression tests
- [ ] Chores
- [x] Documentation changes
- [ ] Features include unit/acceptance tests
- [ ] Refactoring
- [ ] Update dependencies

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -578,7 +578,7 @@ See [Conventional Commits](Https://conventionalcommits.org) for commit guideline
 
 * don't automatically url encode keyset values
 
-* fixed bug wher embedded resources would always provide defaults, not allowing you to declare your own (primary?) actions (#339)
+* fixed bug when embedded resources would always provide defaults, not allowing you to declare your own (primary?) actions (#339)
```
Comment (zachdaniel) [inline CHANGELOG.md:581]: where

File: CHANGELOG.md [hunk 2]
```diff
@@ -319,7 +319,7 @@ See [Conventional Commits](Https://conventionalcommits.org) for commit guideline
 
 ### Bug Fixes:
 
-* propery set authorize?: false on runtime filter
+* property set authorize?: false on runtime filter
```
Comment (zachdaniel) [inline CHANGELOG.md:322]: properly

**Review by zachdaniel — CHANGES_REQUESTED** [review]
some of these need to be updated

**Comment by george124816** [comment]
@zachdaniel done :)

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**